.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "sux"]
```

Comment:
If we do not use submodules, do we still need the .gitmodules file?

Reply:
Removed .gitmodules file